sentry-core/src/main/java/io/sentry/core/SentryThreadFactory.java

```diff
@@ -55,7 +55,9 @@ private SentryThread getSentryThread(
     if (crashedThreadId != null) {
       sentryThread.setCrashed(crashedThreadId == thread.getId());
     }
-    sentryThread.setCurrent(thread == currentThread);
+    sentryThread.setErrored(thread == currentThread);
+    //    sentryThread.setCurrent(thread == currentThread); TODO: do we need that? this should be if
```

Comment:
@bruno-garcia `current` should be if this thread is `foreground` or not, so we will depend on the Android FW.

`Looper.getMainLooper().getThread() == Thread.currentThread()` means `current` = true if I got it right.

Reply:
That means UI threat. Foreground would be isDaemon=false.
Could be useful to also flag the UI threat on mobile I assume

Reply:
Sorry I'm talking about isDaemon which has nothing to do with current. Definition of current is muddy imo. By looking at the name I think of the thread where the error happened. Hence the code. If an exception is thrown in a background there. I'd Mark that one as current and show that on the UI.
I believe we're going for 'erroed' to display in the UI which is fine

Reply:
@bruno-garcia `Foreground would be isDaemon=false` is not correct for Android, that's why we need to compare the `Looper.getMainLooper().getThread()` for Android.

Reply:
I guess the concept doesn't make much sense then since it's a thread that avoid the process from ending. Doesn't make much sense in mobile anyway.on Windows you can keep a process running if u have a foreground thread alive even if the UI thread exited. I assume that doesn't happen in mobile